tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -136,11 +140,10 @@ def test_harvesting_arxiv_workflow_manual_rejected(
     return_value=[],
 )
 def test_harvesting_arxiv_workflow_core_record_auto_accepted(
-    mocked_download,
-    mocked_is_pdf,
```

Comment:
Looks like you removed one too many, strange it did not fail though (checking locally).

Reply:
Tested locally, yes, it's overriding the workflow_app. My guess is that one of the other fixtures (probably mocked_external_services) is already requiring the workflow app, and it's doing so inside an app context, so the magick mock `workflow_app.app_context()` is not really doing anything, and it does not fail just because it's lucky.